File: Java/Animal.java
public class Animal {

    private String raca;
    private String nome;
    private int idade;

    public Animal(String raca, String nome, int idade){
        this.raca = raca;
        this.nome = nome;
        this.idade = idade;
    }

    public String getRaca() {
        return raca;
    }

    public void setRaca(String raca) {
        this.raca = raca;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public void mostrarInformacoes(){
        System.out.println("Raça do animal: "+ this.raca);
        System.out.println("Nome do animal: "+ this.nome);
        System.out.println("Idade do animal: "+ this.idade);
    }

    public void Som(){
        System.out.println("Som do animal.");
    }

}

File: Python/Animal.py
from Cachorro import Cachorro
from Gato import Gato


class Animal:
    def __init__(self, raca, nome, idade):
        self.raca = raca
        self.nome = nome
        self.idade = idade

    def get_raca(self):
        return self.raca

    def set_raca(self, raca):
        self.raca = raca

    def get_nome(self):
        return self.nome

    def set_nome(self, nome):
        self.nome = nome

    def get_idade(self):
        return self.idade

    def set_idade(self, idade):
        self.idade = idade

    def mostrar_informacoes(self):
        print(f"Raça do animal: {self.raca}")
        print(f"Nome do animal: {self.nome}")
        print(f"Idade do animal: {self.idade}")

    def som(self):
        print("Som do animal.")

cachorro = Cachorro("Labrador", "Rex", 5)
gato = Gato("Siamês", "Felix", 3)

cachorro.mostrar_informacoes()
cachorro.som()

gato.mostrar_informacoes()
gato.som()